### PR TITLE
Unify with initdisplay and add gopher-badge and wioterminal

### PR DESCRIPTION
**Pull request by sago35**
In this PR, examples/initdisplay will be used to unify the code.
I also added gopher-badge and wioterminal to confirm the unification.

I have checked with the following boards.

* pyportal
* wioterminal
* gopher-badge

I do not have the following boards and have not been able to check them.

* pybadge
* clue

image

### CODE REVIEW

File: examples/initdisplay/pybadge.go [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+	machine.SPI1.Configure(machine.SPIConfig{
+		SCK:       machine.SPI1_SCK_PIN,
+		SDO:       machine.SPI1_SDO_PIN,
+		SDI:       machine.SPI1_SDI_PIN,
+		Frequency: 8000000,
+	})
+
+	display := st7735.New(machine.SPI1, machine.TFT_RST, machine.TFT_DC, machine.TFT_CS, machine.TFT_LITE)
+	display.Configure(st7735.Config{
+		Rotation: st7735.ROTATION_90,
```
Comment (deadprogram) [inline examples/initdisplay/pybadge.go]: You should change this to  `st7735.ROTATION_180` for it to work as you expect on this device.

**Comment by deadprogram** [comment]
Clue works as expected with this PR.

File: examples/initdisplay/pybadge.go (hunk 1 above)
Comment (sago35) [reply examples/initdisplay/pybadge.go]: Corrected.

**Comment by sago35** [comment]
Please review again. @deadprogram

**Comment by deadprogram** [comment]
Thank you for the improvements here @sago35 now merging.

**Pull request merged**